Interests.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Beyonce Knowles Resume</title>
+  <link rel="stylesheet" type="text/css" href="css/styles.css">
+</head>
+<body>
+
+  <div>
+      <img src="Images/Beyonce_Head.jpg" alt="Beyonce headshot">
```

Comment:
the image is not loading because you are calling `Images/Beyonce_Head.jpg` however your folder structure is `images/Beyonce_Head.jpg` 🔨